LayoutTests/TestExpectations

```diff
@@ -7763,3 +7758,15 @@ imported/w3c/web-platform-tests/css/CSS2/floats/float-nowrap-hyphen-rewind-1.htm
 webkit.org/b/288689 svg/text/bidi-text-query.svg [ Skip ]
 
 webkit.org/b/289782 http/tests/security/contentSecurityPolicy/script-src-strict-dynamic-and-script-src-elem.html [ Pass Failure ]
+
+webkit.org/b/289926 imported/w3c/web-platform-tests/css/css-position/overlay/overlay-transition-backdrop.html [ ImageOnlyFailure ]
+webkit.org/b/289926 imported/w3c/web-platform-tests/css/css-position/overlay/overlay-transition-dialog.html [ ImageOnlyFailure ]
+webkit.org/b/289926 imported/w3c/web-platform-tests/css/css-position/overlay/overlay-transition-finished.html [ ImageOnlyFailure ]
+webkit.org/b/289926 imported/w3c/web-platform-tests/css/css-position/overlay/overlay-transition-in-rendering.html [ ImageOnlyFailure ]
+webkit.org/b/289926 imported/w3c/web-platform-tests/css/css-position/overlay/overlay-transition-out-rendering.html [ ImageOnlyFailure ]
```

Comment:
There's a bug for this: https://bugs.webkit.org/show_bug.cgi?id=276727

Reply:
I changed the link to that bugzilla